src/main/app/forms/models/fileTypeErrors.ts

```diff
@@ -16,7 +17,9 @@ export class FileTypeErrors {
   static all (): FileTypeErrors[] {
     return [
       FileTypeErrors.WRONG_FILE_TYPE,
-      FileTypeErrors.FILE_TOO_LARGE
+      FileTypeErrors.FILE_TOO_LARGE,
+      FileTypeErrors.FILE_REQUIRED,
+      FileTypeErrors.REQUEST_TIMEOUT
```

Comment:
That's not a part of **file type** errors.

Reply:
reworded for clarity